system_tests/queue_system_test.go

```diff
@@ -100,5 +100,7 @@ var _ = Describe("Queue Controller", func() {
 			return err
 		}, 30).Should(HaveOccurred())
 		Expect(err.Error()).To(ContainSubstring("Object Not Found"))
+
+		//FIXME implement delete queue test with ifUnused/ if Empty?
```

Comment:
I don't think we need to. We would be testing RabbitMQ API. From our point of view, it is sufficient to test that we send the adequate request to rabbit; then we rely on rabbit doing the right thing.